lsp-methods.el

```diff
@@ -940,7 +940,15 @@ https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#co
       :annotation-function #'lsp--annotate
       :display-sort-function #'lsp--sort-completions)))
 
-;;; TODO: implement completionItem/resolve
+(defun lsp--resolve-completion (comp)
+  (lsp--cur-workspace-check)
+  (lsp--send-changes lsp--cur-workspace)
+  (unless (gethash "resolveProvider" (lsp--capability "completionProvider"))
+    (error "ResolveProvider not found"))
```

Comment:
hmmm this is not what i meant. The previous version is ok. Just need to move the nullness check of `comp` out of the `resolveProvider` check. So when `comp` is nil, it fails no matter `resolveProvider` is nil or not.